x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/elser/ElserMlNodeServiceSettingsTests.java

```diff
@@ -93,15 +93,9 @@ public void testTransportVersionIsCompatibleWithElserModelVersion() {
                 TransportVersions.ELSER_SERVICE_MODEL_VERSION_ADDED
             )
         );
-        assertTrue(
-            ElserMlNodeServiceSettings.transportVersionIsCompatibleWithElserModelVersion(
-                TransportVersions.ELSER_SERVICE_MODEL_VERSION_ADDED_PATCH
-            )
-        );
+        assertTrue(ElserMlNodeServiceSettings.transportVersionIsCompatibleWithElserModelVersion(TransportVersions.V_8_11_X));
 
-        assertFalse(
-            ElserMlNodeServiceSettings.transportVersionIsCompatibleWithElserModelVersion(TransportVersions.ML_PACKAGE_LOADER_PLATFORM_ADDED)
-        );
```

Comment:
Nice spotting this! It was actually the wrong version, but since it was an assertFalse we realized this only when we collapsed - it is "V_8_10_X", right?

Reply:
Yeah, this PR doesnt have the rename (yet)